flipperdevices/flipperzero-firmware
# Infrared: Add Olimpia Splendid AC

# What's new

- New AC for universal remote

# Verification 

- Upload universal remote database to flipper
- Verify aircon universal remote

# Checklist (For Reviewer)
